internal/pkg/cli/svc_init.go

```diff
@@ -420,6 +432,21 @@ func parseHealthCheck(df dockerfileParser) (*manifest.ContainerHealthCheck, erro
 	}, nil
 }
 
+func (o *initSvcOpts) dockerPlatform() (os, arch string, err error) {
```

Comment:
This method is duplicated right? What do you think of refactoring it into a function:

```go
func dockerPlatform(engine dockerEngine) (os, arch string, err error)
```